Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -384,4 +356,269 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
     }
 
     return lines;
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _DWriteGetFontFamilyNames() {
+    NSMutableArray<NSString*>* fontFamilyNames = [NSMutableArray<NSString*> array];
+
```

Comment:
I do not think moving to CF is a pressing requirement now. We have a lot more work to get CT standing before diverting our focus to this

---
In reply to: [78611079](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78611079) [](ancestors = 78611079,78610729,78225402,78221279,78220974)